projects/ion/src/lib/core/types/context.ts

```diff
@@ -0,0 +1 @@
+export type Context = 'success' | 'info' | 'warning' | 'negative' | 'neutral';
```

Comment:
You can add this in status.ts like TagStatusType;

export type TagStatusType = 'success' | 'info' | 'warning' | 'negative' | 'neutral';